### PR TITLE
fix: dont delay loading before keplr anymore

### PR DESCRIPTION
**Pull request by samsiegart**

### CODE REVIEW

File: wallet/public/index.html [hunk 1]
```diff
@@ -3,6 +3,28 @@
...
     <script src="%PUBLIC_URL%/lockdown.umd.js"></script>
+    <script>
+      const devUnsafe = '%NODE_ENV%' === 'development' ? 'unsafe' : 'safe';
+
+      const { pow: mathPow } = Math;
+      Math.pow = (base, exp) =>
+        typeof base === 'bigint' && typeof exp === 'bigint'
+          ? base ** exp
+          : mathPow(base, exp);
+
```
Comment (turadg) [inline wallet/public/index.html:14]: These are sort of fresh lines. Might want to omit https://github.com/Agoric/wallet-app/issues/84 
```suggestion
```

File: wallet/public/index.html [hunk 2]
```diff
@@ -31,93 +53,22 @@
...
-    <% }); %>
   </head>
-
   <body style="overflow-x: hidden; overflow-y: scroll">
-    <%= htmlWebpackPlugin.tags.bodyTags %>
     <noscript>You need to enable JavaScript to run this app.</noscript>
-    <div id="root">
-      <!-- replaced by React mounting -->
-    </div>
-    <div id="keplr-missing" hidden>
```
Comment (turadg) [inline wallet/public/index.html:111]: what will happen when the Keplr extension isn't installed?

File: wallet/public/index.html (hunk 1 above)
Comment (samsiegart) [reply wallet/public/index.html:14]: Sure, done. To be clear, in this PR I just restored this file from before the recent keplr change, and then changed taming to default to safe per @erights previous comments.

File: wallet/public/index.html (hunk 2 above)
Comment (samsiegart) [reply wallet/public/index.html:111]: It will error, same behavior as before

**Review by turadg — APPROVED** [review]

**Pull request merged**